include/redis3m/connection.h

```diff
@@ -27,6 +28,20 @@ class connection: utils::noncopyable
     typedef std::shared_ptr<connection> ptr_t;
 
     /**
+     * @brief Create and open a new connection with Timeout
+     * @param host hostname or ip of redis server, default localhost
+     * @param port port of redis server, default: 6379
+     * @param timeval timeout interval struct, default: 1 second
+     * @return
+     */
+    inline static ptr_t createTimeout(const std::string& host="localhost",
+                               const unsigned int port=6379,
+                               const struct timeval timeout=defaultTimeout())
```

Comment:
For consistency with the rest of the project, I'd like `create_timeout` instead of `createTimeout`. So underscore syntax. Is it a problem for you?